app/models/text_component.rb

```diff
@@ -3,7 +3,9 @@ class TextComponent < ActiveRecord::Base
   has_and_belongs_to_many :triggers
   has_many :sensors, through: :triggers
   has_many :events, through: :triggers
+  has_and_belongs_to_many :channels
   belongs_to :report
+  belongs_to :publishing_context
```

Comment:
I thought we could define many keywords in the ChatFuel interface? Those keywords can all be mapped to one channel